app/root/root.css

```diff
@@ -824,6 +824,25 @@ code .comment {
   margin-left: 0;
 }
 
+.titles .role-badge {
+  margin-right: 8px;
+}
+
+.role-badge {
+  border-radius: 12px;
+  padding: 0 8px;
```

Comment:
how does this look with like 2px of vertical padding?

Reply:
Looks good: 
![image](https://user-images.githubusercontent.com/2414826/124163579-afc2c600-da6d-11eb-9d0d-b25114d8c849.png)